products/sle/main.pm

```diff
@@ -659,7 +659,6 @@ sub load_patching_tests {
         if (get_var('LOCK_PACKAGE') && !installzdupstep_is_applicable) {
             loadtest 'console/lock_package';
         }
-        loadtest 'migration/remove_ltss';
```

Comment:
Please also remove file `tests/migration/remove_ltss.pm`

Reply:
Ok, removed it.